### PR TITLE
Switches this gem connect via http to the microservice

### PR DESCRIPTION
**Pull request by jspies**
Instead of straight to redis.

### CODE REVIEW

**Comment by sdrioux** [comment]
🚢 , should we get this repo on Circle?

**Comment by jspies** [comment]
we should

**Comment by jspies** [comment]
done

File: README.md [hunk 1]
```diff
@@ -18,10 +22,10 @@ And then execute:
 
 ## Usage
 
-Set an environment variable FLIPPY_REDIS_URL to connect to your redis machine. If you do not set a variable it defaults to redis://localhost:6379/floppy_db.
+Set an environment variable FEATURE_FLIP_URL to connect to your redis machine. If you do not set a variable it defaults to http://localhost:9292.
```
Comment (jcderr) [inline README.md]: Should this line say something other than `to your redis machine`?

Comment (jspies) [inline README.md]: YES! I should update the readme. I'm just so happy someone looked at this PR

**Comment by sdrioux** [comment]
Fix yo specs!

**Comment by jspies** [comment]
they fixed

**Comment by sdrioux** [comment]
🚢

**Comment by sdrioux** [comment]
:shipit:

**Comment by txssseal** [comment]
screen shot 2016-04-29 at 9 22 25 am

**Comment by jspies** [comment]
@txssseal I think this image is amazing but I don't understand the relevance to this pull request.

**Pull request merged**